docker/README.md

```diff
@@ -70,3 +70,44 @@ After build finished, run the container:
 ```
 docker run --rm -it lightgbm
 ```
+
+## Running the R-package Сontainer
+
+Build the container based on the [`verse` Rocker image](https://www.rocker-project.org/images/), for R users:
+
+```
+mkdir lightgbm-docker
+cd lightgbm-docker
+wget https://raw.githubusercontent.com/Microsoft/LightGBM/master/docker/dockerfile-r
+docker build -t lightgbm-r -f dockerfile-r .
+```
+
+After the build is finished you have two options to run the container:
+
+1. Start [RStudio](), an interactive development environment, so that you can develop your analysis using LightGBM or simply try out the R package.
```

Comment:
Did you accidently forget a link here `[RStudio]()` or was it just to emphasize the word? If latter, please use another way to do that.